jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/HandlerRegistry.java

```diff
@@ -50,6 +55,7 @@ static HandlerRegistry createDefault(MeterProvider meterProvider) {
     var seen = new HashSet<String>();
     for (var bean : ManagementFactory.getGarbageCollectorMXBeans()) {
       var name = bean.getName();
+      garbageCollectors.add(name);
```

Comment:
It's pretty tricky to tell which handlers get added in response to which garbage collectors. How about just a switch statement with cases for each known garbage collector bean name, where each case explicitly registers its required handlers?

Reply:
I think that's a good idea. Implemented suggestion.